### PR TITLE
[DM-3100] Shell device tab - command name and description should be truncated

### PR DESCRIPTION
**Pull request by catarino-sag**

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-10-18-503-0-shell-device-tab-command-name-and-description-should-be-truncated.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3100
+version: 10.18.503.0
+---
+When adding a new predefined command on Shell device tab, all text fields are now properly formatted making all information visible.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-10-18-503-0-shell-device-tab-command-name-and-description-should-be-truncated.md:17]:
```suggestion
When adding a new predefined command on the **Shell** tab in the device details, all text fields are now properly formatted making all information visible.
```

File: content/change-logs/device-management/ui-c8y-10-18-503-0-shell-device-tab-command-name-and-description-should-be-truncated.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: 
+title: Shell device tab - command name and description should be truncated
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-10-18-503-0-shell-device-tab-command-name-and-description-should-be-truncated.md:3]: Input in text fields on the Shell tab no longer truncated

Comment (catarino-sag) [reply content/change-logs/device-management/ui-c8y-10-18-503-0-shell-device-tab-command-name-and-description-should-be-truncated.md:3]:
```suggestion
title: Input in text fields on the Shell tab no longer truncated
```

**Comment by BeateRixen** [comment]
@catarino-sag I suggest to change the file name according to the title changes as well as they are quite different now

**Comment by catarino-sag** [comment]
> @catarino-sag I suggest to change the file name according to the title changes as well as they are quite different now

Agreed. fixed.

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Comment by BeateRixen** [comment]
@catarino-sag If this is supposed to go into 2024 release please create a second PR and change the version accordingly.

**Comment by BeateRixen** [comment]
If this change has been added to the release 2024 as well, then you must also create a release notes entry in the release/y2024 branch in this repo. But with a different version number, that is, the one for 2024. But I don´t know if this change is relevant for 2024.

**Pull request merged**